### DISCUSSION TITLE
ContextMenu options

### DISCUSSION THREAD

**Discussion by NightmareBlocks** [Raphire/Win11Debloat#77] [Q&A]
Whatdoes disable context menu options mean?

**Comment by Raphire** [comment]
The context menu is also known as the right-click menu. 

![PDnU4EyKFx9aHhCHXXRzu7](https://github.com/Raphire/Win11Debloat/assets/9938813/e56a67d4-feba-422b-b9b9-a36390efd984)

Win11Debloat has the option to remove a few of these options to clean up the menu a bit. The script only gives you the option to disable these if you're running Windows 10 or if you select the option to revert the Windows 11 Context menu to the old Windows 10 style. 

I don't really recommend removing these context menu options, as in my experience they are brought back after an update.

**Reply by NightmareBlocks** [reply]
Thanks! There are some things I know about Windows, that you really dont need, and others like this one... yeah...
